addons/mail/static/src/components/thread_preview/thread_preview.js

```diff
@@ -37,15 +37,25 @@ export class ThreadPreview extends Component {
     }
 
     /**
-     * Get inline content of the last message of this conversation.
+     * Get content of the last message, including inlineLastMessageBody or lastTrackingValue.
      *
-     * @returns {string}
+     * @returns {object}
      */
-    get inlineLastMessageBody() {
+    get lastMessageContent() {
+        const messageContent = {}
         if (!this.threadPreviewView.thread.lastMessage) {
-            return '';
+            return messageContent;
+        }
+        if (this.threadPreviewView.thread.lastMessage.lastTrackingValue) {
+            const trackingValue = this.threadPreviewView.thread.lastMessage.lastTrackingValue;
+            messageContent.trackingValue = {
+                changedFieldAsString : trackingValue.changedFieldAsString,
+                oldValueAsString : trackingValue.oldValueAsString,
+                newValueAsString : trackingValue.newValueAsString,
+            }
         }
-        return mailUtils.htmlToTextContentInline(this.threadPreviewView.thread.lastMessage.prettyBody);
+        messageContent.body = mailUtils.htmlToTextContentInline(this.threadPreviewView.thread.lastMessage.prettyBody);
+        return messageContent;
```

Comment:
Turn this into field(s), and ideally field is not object but instead record or primitives.